lib/modules/datasource/sbt-plugin/index.ts

```diff
@@ -9,10 +9,9 @@ import { SbtPackageDatasource } from '../sbt-package';
 import { getLatestVersion, parseIndexDir } from '../sbt-package/util';
```

Comment:
> Change the default repository for SBT plugin lookup to https://repo1.maven.org/maven2/

**Maintainers**: is this a `fix:` or `feat:` type change. It feels more like a `feat:` or maybe even a breaking feature change?

Unless the entire thing is broken right now because everybody migrated to the new stuff and Renovate is still using the old defaults. In that case it's probably a `fix:` type change.

Reply:
I believe it's a `fix`. We run self-hosted Renovate and we find that sbt plugins have not been updated for some time. Recently we configured our `sbt-plugins` to use `https://repo1.maven.org/maven2/` and now we are getting some plugin updates.

According to [this page](https://www.scala-sbt.org/1.x/docs/Bintray-For-Plugins.html) (emphasis mine):

>As of 2021-04-18, we have migrated all sbt plugins and sbt 0.13 artifacts to the Artifactory instance, and redirected https://repo.scala-sbt.org/scalasbt/ to point to it as well, so existing builds should continue to work without making any changes today and after May 1st. **For plugin hosting, we will operate this as a read-only repository. Any new plugin releases should migrate to using [Sonatype OSS](https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html)**.

So I think Renovate has not been updating sbt plugins since 2021, because it's been looking at a read-only mirror that was frozen in 2021 (`https://repo.scala-sbt.org/scalasbt/`).

When I go to Sonatype OSS, their releases repo `https://oss.sonatype.org/content/repositories/releases/` just redirects to `https://repo1.maven.org/maven2/`, so that seems like the right thing to configure.